src/common/components/repository-row/index.js

```diff
@@ -135,6 +154,17 @@ class RepositoryRow extends Component {
     let caption;
     if (registerNameStatus.success) {
       caption = <div>{ tickIcon } Registered successfully</div>;
+    } else if ( registerNameStatus.error
+      && registerNameStatus.error.json.payload.code === 'already_registered') {
```

Comment:
@bartaz I need to look for more error codes from SCA this morning, one I've seen that is unhandled is 'unauthorized' when we're signed out of SSO (expired during session)